mardi_importer/mardi_importer/zbmath/ZBMathPublication.py

```
@@ -142,6 +195,12 @@ def insert_claims(self):
             self.item.add_claims(classification_claims)
         if self.de_number:
             self.item.add_claim(self.label_id_dict["de_number_prop"], self.de_number)
+        if self.licenses:
+            license_claims = []
+            for l in self.licenses:
```

Ruff (0.15.9)

[error] Ambiguous variable name: `l`

(E741)